### PR TITLE
Delete Garbage

### PR DESCRIPTION
**Pull request by ronyjacobson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,2 @@
 # Hello-world
 My first repo!
-sdfdf
-dfgsdgf
```
Comment (rjl10) [inline README.md]: tgtt

File: README.md [hunk 2]
```diff
@@ -1,5 +1,2 @@
 # Hello-world
 My first repo!
```
Comment (rjl10) [inline README.md]: repo->repository

Comment (ronyjacobson) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (ronyjacobson) [inline README.md]: Done

File: Hi this is my solution.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Hi this is my solution
```
Comment (rjl10) [inline Hi this is my solution.txt]: Hi please fix typo

File: Hi this is my solution.txt [hunk 4]
```diff
@@ -1 +1 @@
-Hi this is my solution
\ No newline at end of file
+Hi this is my solution.
```
Comment (rjl10) [inline Hi this is my solution.txt]: Please remove dot at the end

Comment (ronyjacobson) [inline Hi this is my solution.txt]: Done

**Pull request merged**